src/main/java/me/totalfreedom/totalfreedommod/command/Command_tag.java

```diff
@@ -15,7 +15,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.OP, source = SourceType.BOTH)
-@CommandParameters(description = "Allows you to set your own prefix.", usage = "/<command> [-s[ave]] <set <tag..> | list | gradient <hex> <hex> <tag..> | off | clear <player> | clearall>")
```

Comment:
Any specific reason for dropping -save?

Reply:
most, if not all people use -s and not -save, and it also looks dumb in the syntax imo